src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -303,38 +281,13 @@ uct_dc_mlx5_ep_short_dm(uct_dc_mlx5_ep_t *ep, uct_rc_mlx5_dm_copy_data_t *cache,
         return status;
     }
 
-    uct_dc_mlx5_iface_bcopy_post(iface, ep, opcode,
-                                 hdr_len + length,
-                                 rdma_raddr, rdma_rkey,
-                                 desc, fm_ce_se, 0, buffer,
-                                 log_sge.num_sge ? &log_sge : NULL);
-    return UCS_OK;
-}
-
-static ucs_status_t UCS_F_ALWAYS_INLINE uct_dc_mlx5_ep_short_iov_dm(
-        uct_dc_mlx5_ep_t *ep, uct_rc_mlx5_dm_copy_data_t *cache, size_t hdr_len,
-        const uct_iov_t *iov, size_t iovcnt, size_t iov_length)
-{
-    uct_dc_mlx5_iface_t *iface     = ucs_derived_of(ep->super.super.iface,
-                                                    uct_dc_mlx5_iface_t);
-    uct_rc_iface_send_desc_t *desc = NULL;
-    /* pass dummy pointer as 0-length payload to avoid static check issue */
-    char dummy                     = 0;
-    void *buffer;
-    ucs_status_t status;
-    uct_ib_log_sge_t log_sge;
-
-    status = uct_rc_mlx5_common_dm_make_data(&iface->super, cache, hdr_len,
-                                             &dummy, 0, iov, iovcnt, &desc,
-                                             &buffer, &log_sge);
-    if (ucs_unlikely(UCS_STATUS_IS_ERR(status))) {
-        return status;
-    }
-
-    uct_dc_mlx5_iface_bcopy_post(
-            iface, ep, MLX5_OPCODE_SEND, hdr_len + iov_length, 0, 0, desc,
-            MLX5_WQE_CTRL_SOLICITED | MLX5_WQE_CTRL_CQ_UPDATE, 0, buffer,
-            (log_sge.num_sge > 0) ? &log_sge : NULL);
+    UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
```

Comment:
can we also unite the "uct_dc_mlx5_ep_short_dm" function ?

Reply:
With `uct_rc_mlx5_ep_short_dm`?

Reply:
@yosefe I've combined `uct_dc_mlx5_ep_short_dm` and `uct_rc_mlx5_ep_short_dm` to `uct_rc_mlx5_common_ep_short_dm`